docs/authentication/enterprise-connections/saml/azure.mdx

```diff
@@ -17,39 +17,60 @@ description: Learn how to allow users to sign up and sign in to your Clerk app w
     }
   ]}
 >
-  - Use Microsoft Azure Entra ID to enable SSO via SAML for your Clerk app
+  - Create a SAML SSO connection for your customer
```

Comment:
To prepare for the self-serve world - app owner reading this doc to customer SSO into Clerk dashboard, what do you think of removing "your customer" here

```suggestion
  - Create a SSO connection via Clerk Dashboard 
```